ecs-cli/modules/commands/flags.go

```diff
@@ -77,13 +79,30 @@ const (
 	RoleFlag                                = "role"
 )
 
+// OptionalClusterAndProfileFlags provides these flags:
 // OptionalClusterFlag inline overrides cluster
-func OptionalClusterFlag() cli.Flag {
-	return cli.StringFlag{
-		Name: ClusterFlag + ", c",
-		Usage: fmt.Sprintf(
-			"[Optional] Specifies the ECS cluster name to use. Defaults to the cluster configured using the configure command",
-		),
+// OptionalClusterConfigFlag specifies the cluster profile to read from config
+// OptionalProfileConfigFlag specifies the credentials profile to read from the config
```

Comment:
Question mostly for just my own understanding -- do cluster and profile flags have to be coupled? As in, do we always expect a profile flag if cluster flag is specified (and vice versa)?